packages/shared_preferences/shared_preferences/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 2.3.3
 
 * Clarifies scope of prefix handling in README.
+* Adds description for isValidKey method in shared_preferences_async
```

Comment:
this will need a new version number `2.3.4` here and in the pubspec file

Reply:
Done!